packages/notebook/src/widget.ts

```diff
@@ -582,30 +590,13 @@ export class StaticNotebook extends WindowedList {
             this.model!.cells.length + delta.delete,
             -1 * delta.delete
           );
-          // Add default cell if there are no cells remaining.
-          // @todo this should probably be handled by shared-notebook
-          // @todo this is duplicatively implemented (see other occurrences of notebookconfig.defaultCell)
-          if (!sender.cells.length) {
-            const model = this.model;
-            // Add the cell in a new context to avoid triggering another
-            // cell changed event during the handling of this signal.
-            requestAnimationFrame(() => {
-              if (
-                model &&
-                !model.isDisposed &&
-                !model.sharedModel.cells.length
-              ) {
-                model.sharedModel.insertCell(
-                  0,
-                  sharedModels.createCell({
-                    cell_type: this.notebookConfig.defaultCell
-                  })
-                );
-              }
-            });
-          }
         }
       });
+
+      if (!this.model!.sharedModel.cells.length) {
+        (this.layout as NotebookWindowedLayout).header =
+          this._emptyNotebookMessage;
```

Comment:
Probably better to call a protected method `addHeader` in which you instantiate the information widget.